acceptance/image/image.go

```diff
@@ -633,17 +633,14 @@ func JSONAttestationSignaturesFrom(ctx context.Context) (map[string]string, erro
 
 	signatures := map[string]string{}
 	for name, signature := range state.AttestationSignatures {
-		json, err := json.Marshal(signature)
-		if err != nil {
-			return nil, err
-		}
-		signatures[name] = string(json)
+		signatures[fmt.Sprintf("%s_KEY_ID_%s", prefix, name)] = signature.KeyID
+		signatures[fmt.Sprintf("%s_%s", prefix, name)] = signature.Signature
```

Comment:
Seems that `ATTESTATION_SIGNATURE_...` can be `""`, then in snaps.go we replace everything with `${ATTESTATION_SIGNATURE_...}`, so let's guard here

```suggestion
		if signature.KeyID != "" {
			signatures[fmt.Sprintf("%s_KEY_ID_%s", prefix, name)] = signature.KeyID
		}
		if signature.Signature != "" {
			signatures[fmt.Sprintf("%s_%s", prefix, name)] = signature.Signature
		}
```